src/redux/sagas/actions/removeVerifiedMembers.ts

```diff
@@ -0,0 +1,152 @@
+    annotationMessage,
+  },
+  meta: { id: metaId, navigate, setTxHash },
+  meta,
+}: Action<ActionTypes.ACTION_REMOVE_VERIFIED_MEMBERS>) {
+  let txChannel;
+
+  try {
+    if (!colonyAddress) {
+      throw new Error(
+        'No colony address set for removeVerifiedMembers transaction',
+      );
+    }
+    if (!Array.isArray(members) || members.length === 0) {
+      throw new Error('No members set for removeVerifiedMembers transaction');
+    }
+
+    txChannel = yield call(getTxChannel, metaId);
+
+    // setup batch ids and channels
+    const batchKey = 'removeVerifiedMembers';
+
+    const { removeVerifiedMembers, annotateRemoveVerifiedMembers } =
+      yield createTransactionChannels(metaId, [
+        'removeVerifiedMembers',
+        'annotateRemoveVerifiedMembers',
+      ]);
+
+    yield fork(createTransaction, removeVerifiedMembers.id, {
+      context: ClientType.ColonyClient,
+      methodName: 'editColonyByDelta',
+      identifier: colonyAddress,
+      params: [],
+      group: {
+        key: batchKey,
+        id: metaId,
+        index: 0,
+      },
+      ready: false,
+    });
+
+    if (annotationMessage) {
+      yield fork(createTransaction, annotateRemoveVerifiedMembers.id, {
+        context: ClientType.ColonyClient,
+        methodName: 'annotateTransaction',
+        identifier: colonyAddress,
+        params: [],
+        group: {
+          key: batchKey,
+          id: metaId,
+          index: 1,
+        },
+        ready: false,
+      });
+    }
+
+    yield takeFrom(
+      removeVerifiedMembers.channel,
+      ActionTypes.TRANSACTION_CREATED,
+    );
+    if (annotationMessage) {
+      yield takeFrom(
+        annotateRemoveVerifiedMembers.channel,
+        ActionTypes.TRANSACTION_CREATED,
+      );
+    }
+
+    // add params, how to blobify the last JSON??
+    yield put(
+      transactionAddParams(removeVerifiedMembers.id, [
+        JSON.stringify(
+          getRemoveVerifiedMembersOperation(colonyAddress, members),
+        ),
+      ]),
+    );
+
+    yield initiateTransaction({ id: removeVerifiedMembers.id });
+
+    const {
+      payload: { hash: txHash },
+    } = yield takeFrom(
+      removeVerifiedMembers.channel,
+      ActionTypes.TRANSACTION_HASH_RECEIVED,
+    );
+
+    setTxHash?.(txHash);
+
+    waitForTxResult(removeVerifiedMembers.channel);
+
+    yield createActionMetadataInDB(txHash, customActionTitle);
+
+    if (annotationMessage) {
+      yield uploadAnnotation({
+        txChannel: annotateRemoveVerifiedMembers,
+        message: annotationMessage,
+        txHash,
+      });
+    }
+
+    yield put<AllActions>({
+      type: ActionTypes.ACTION_REMOVE_VERIFIED_MEMBERS_SUCCESS,
+      payload: {},
+      meta,
+    });
+
+    if (colonyName && navigate) {
+      navigate(`/${colonyName}?tx=${txHash}`, {
+        state: { isRedirect: true },
+      });
+    }
+  } catch (error) {
+    return yield putError(ActionTypes.ACTION_DOMAIN_CREATE_ERROR, error, meta);
```

Comment:
I had the same typo in the `add` saga, we need `ACTION_REMOVE_VERIFIED_MEMBERS_ERROR` here not the domain one :D

Reply:
Good catch, thank you! All fixed up.